client/lib/analytics/statsd.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import startsWith from 'lodash/startsWith';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+
+export function statsdUrl( path, eventType, duration, currentPath = undefined ) {
```

Comment:
@ehg Yeah... I don't like it very much. The problem is I don't think it has any test coverage, and so I'm scared to change it! 😬 

I pushed an alternative implementation. Basically, I'm leaving the hacky bit in `client/lib/analytics/index.js` where it originally was. So now `statsdTimingUrl` just takes a `featureSlug`, `eventType`, and `duration`. This should work, because from the server side I don't think we have much reason (currently) to pass in a `path`.

WDYT?